clients/vscode-hlasmplugin/src/hlasmExternalFiles.ts

```diff
@@ -17,6 +17,10 @@ import * as vscodelc from "vscode-languageclient";
 import { isCancellationError } from "./helpers";
 import { uriFriendlyBase16Decode, uriFriendlyBase16Encode } from "./conversions";
 
+import * as crypto from "crypto";
```

Comment:
Will this work in browser? There is https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto that should work both in node an browser. Example of doing sha256 [here](https://developer.mozilla.org/en-US/docs/Web/API/Web_Crypto_API/Non-cryptographic_uses_of_subtle_crypto)

Reply:
Btw, based on a quick research, the subtlecrypto api has been hopelessly broken first by Google and then by W3C. So it probably won't work in browser and we cannot use the suggested api anyway.